operator/src/main/resources/scripts/introspectDomain.py

```diff
@@ -930,7 +930,7 @@ def __init__(self, env):
         self.macroStr+=', '
       self.macroStr+='${' + key + '}'
 
-    trace("available macros: '" + self.macroStr + "'")
+    trace("Available macros: '" + self.macroStr + "'")
```

Comment:
Hmmm... okay, I guess we don't have any of the introspector internationalized. :(

Reply:
Yah. You and I lowered the priority of doing that type of work a week or two ago, only relatively 'P1' priority stuff is in this check-in (see OWLS-74901).